pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -8,7 +8,7 @@ ARG CUBLAS_VERSION=11.10.3.66
 ARG CUDNN_VERSION=8.5.0.96
 
 # PyTorch Binaries and versions.
-ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.13.0_ec2/20221105-023216/41160d617a294a8963a868ec942a2569a2fbf3ea/torch-1.13.0%2Bcu117-cp39-cp39-linux_x86_64.whl
+ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.13.0_ec2/20221205-210214/8cf843550346728d7e661cd0a26f74b603dc17a5/torch-1.13.0%2Bcu117-cp39-cp39-linux_x86_64.whl
```

Comment:
The binary name says `r1.13.0_ec2`. Does it just correspond to `ec2` images or `SM` as well?

Reply:
Inference  images only use one type of binary, being EC2, for both EC2 and SageMaker. Training images need a separate  binary for SageMaker.